app/src/main/java/eu/kanade/tachiyomi/ui/setting/SettingsReaderController.kt

```diff
@@ -319,12 +319,13 @@ class SettingsReaderController : SettingsController() {
                 titleRes = R.string.pref_webtoon_side_padding
                 entriesRes = arrayOf(
                     R.string.webtoon_side_padding_0,
+                    R.string.webtoon_side_padding_5,
```

Comment:
this actually needs to be added in the strings xml too

Reply:
@Jays2Kings Do I need to manually add the missing stings in all languages

Reply:
Just the main strings xml

Reply:
Added the missing percentage string to mail strings.xml